<ui version="4.0" >
 <class>ScriptEditWidget</class>
 <widget class="QWidget" name="ScriptEditWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="5" >
    <widget class="Lua::LuaEditor" name="m_scriptEdit" >
     <property name="tabStopWidth" >
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="m_runButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>run</string>
     </property>
     <property name="shortcut" >
      <string>Ctrl+Return</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLineEdit" name="m_filenameEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QToolButton" name="m_openButton" >
     <property name="toolTip" >
      <string>open</string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../qosmic.qrc" >
       <normaloff>:/icons/silk/folder.xpm</normaloff>:/icons/silk/folder.xpm</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QToolButton" name="m_saveButton" >
     <property name="toolTip" >
      <string>save</string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../qosmic.qrc" >
       <normaloff>:/icons/silk/disk.xpm</normaloff>:/icons/silk/disk.xpm</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="m_stopButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>stop</string>
     </property>
     <property name="shortcut" >
      <string>Ctrl+Shift+Return</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Lua::LuaEditor</class>
   <extends>QTextEdit</extends>
   <header>src/lua/luaeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qosmic.qrc" />
 </resources>
 <connections/>
</ui>
